bindings/matrix-sdk-ffi/src/authentication.rs

```diff
@@ -171,7 +171,7 @@ impl OidcConfiguration {
                 .iter()
                 .filter_map(|(issuer, client_id)| {
                     let Ok(issuer) = Url::parse(issuer) else {
-                        tracing::error!("Failed to parse {:?}", issuer);
+                        tracing::error!("Failed to parse {issuer:?}");
```

Comment:
Is there a particular reason to prefer one format over the other here, out of interest?